lib/3_add_up_numbers.php

```diff
@@ -8,8 +8,32 @@
 
 function addUpNumbers($arr) {
   // Your code goes here.
+	$sum = array_sum($arr);
+	return $sum;
 }
 
 // Write your own "tests" below. Refer to the the examples from exercise #1.
+echo "\nTesting that the sum of numbers in $number1 returns 12.\n";
+$number1 = array(1, 1, 6, 2, 2);
+if (addUpNumbers($number1) == 12) {
+  echo "Success!";
+}
+else{
+  echo "Failed!";
+}
+echo "\n-------------------------------------------------------------------\n";
+
+echo "\nTesting that the sum of numbers in $number2 returns 27.\n";
+$number2 = array(5, 8, 10, 4);
+if (addUpNumbers($number2) == 27) {
```

Comment:
Nice! I don't see this test scenario being meaningfully different than the first. Other scenarios you might consider testing: Does your method work for floating point numbers? A combination of integers and floating point? Negative numbers?